src/middlewared/middlewared/plugins/system/info.py

```diff
@@ -130,6 +130,7 @@ async def build_time(self, app):
         return datetime.fromtimestamp(int(buildtime)) if buildtime else buildtime
 
     @accepts()
+    @no_authz_required
```

Comment:
Is this all really needed? Things like loadavg/uptime can be seen as sort of private.